tutor

```diff
@@ -0,0 +1,2 @@
+Dodaje jakis tekst.'A' dodaje znak na koncu linii.
```

Comment:
It looks like missing vimtutor file

Reply:
I have to point something. In fact, in entire vimtutor, there is no single place, that actually said, that we neet to save "vimtutor" What we really should do is to create filename named "tutor" and put some text in it. I have actually save vimtutor just because it was in the task explanation, and I have done it in spite of vimtutor ;d So it can be misleading.

Reply:
I think It says everything: "Zapisz przerobiony samouczek do pliku o nazwie tutor w swoim repo resolvable-conflict na gałęzi master." ;)

Reply:
You're right, but this way user should "know" `:wq` before it is actually in the tutor. I do not say that you do not have right guys, but in the other hand, you can end up vimtutor without saving it - especially that you have to quit it few times (without saving), so to end this task you have to "go back" just to fulfill the requirement of the task.

Reply:
Ok, now I've got your way of thinking. I have no doubts that you've done this tutorial. ;)